docs/backup-restore.md

```diff
@@ -175,3 +175,56 @@ Always make sure your backup strategy is tested and fits your needs.
 No one solution fits all, and we tried to cover the basic cases.
 For the latest updates, always feel free to ask on the BTCPay Server community channels.
 :::
+
+## How to migrate to a new server?
+
+The necessary steps always depend on your setup and infrastructure but as a rough guideline the following steps and considerations will get you through the process.
+
+:::warning
+If you have Lightning Network (LN) channels open on your internal node, you need to make sure to **not** restore any old channel state, otherwise your are at the risk of loosing funds. Therefore it is important to shut down BTCPay on the old server immediately after the backup has finished as outlined below.
+:::
+
+
+### Preparation   
+
+#### Temporarily reduce the TTL of your domain:   
+Depending on the TTL (time-to-live) of your DNS entries (A-Record) for you BTCPay Server domains/subdomains it makes sense to reduce the TTL a day or a few hours before you want to start the migration. In case anything goes wrong you can quickly switch between new and old server IP addresses. You should set the TTL for the domain/subdomain to some shorter timewindow eg. 600 seconds (10 minutes) or if possible 300 seconds (5 mins). You can do this in at your domain registrar control center.
```

Comment:
```suggestion
Depending on the TTL (time-to-live) of your DNS entries (A-Record) for you BTCPay Server domains/subdomains it makes sense to reduce the TTL a day or a few hours before you want to start the migration.
In case anything goes wrong you can quickly switch between new and old server IP addresses.
You should set the TTL for the domain/subdomain to some shorter time frame eg. 600 seconds (10 minutes) or if possible 300 seconds (5 mins).
You can do this in at your domain registrar control center.
```